core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -21,13 +20,9 @@ import {
   RenderingStrategy,
   SsrOptimizationOptions,
   defaultSsrOptimizationOptions,
+  getDefaultRenderKey,
 } from './ssr-optimization-options';
 
-/**
- * Returns the full url for the given SSR Request.
- */
-export const getDefaultRenderKey = getRequestUrl;
-
```

Comment:
`query:` if we now the default value of `renderKeyResolver`, maybe we can adjust `getRenderingKey` method so it simply uses the value from ssr options?

Reply:
I'm afraid our current (wrong) TS typing of `this.ssrOptions?` (which TS see as possibly-undefined) and `this.ssrOptions.renderKeyResolver?` (which TS see as possibly-undefined) doesn't allow for it.

Reply:
Yes, I faced this issue before when using `ssrOptions` in this class. Let's consider refactoring it in the future 👍